packages/ui/src/components/StatusLegend/StatusLegend.module.css

```diff
@@ -6,18 +6,55 @@
   text-transform: uppercase;
 }
 
-.legend > li:before {
+.legend > li {
+  display: flex;
+  align-items: center;
+  border-bottom: 2px solid transparent;
+}
+
+.legend > li.isSelected {
+  border-color: var(--status-menu-active-border);
+  color: var(--status-menu-active-text);
+  font-weight: 500;
+}
+
+.legend > li > a {
+  text-decoration: none;
+  margin: 0 0 -2px;
+  padding: 0.5em 1em;
+  color: var(--status-menu-text);
+  transition: border-bottom-color 200ms ease-in-out, color 0.1s ease;
+  display: flex;
+  align-items: center;
+}
+
+.legend > li > a:before {
   content: '';
   background-color: var(--item-bg);
   border-radius: 50%;
   width: 0.5rem;
   height: 0.5rem;
-  display: inline-block;
+  display: flex;
```

Comment:
why `flex` is here?

Reply:
You're right, this is not necessary anymore. I think I added it before changing the markup of the filters.